allennlp/commands/dry_run.py

```diff
@@ -87,14 +83,12 @@ def dry_run_from_params(params: Params, serialization_dir: str) -> None:
     prepare_environment(params)
 
     vocab_params = params.pop("vocabulary", {})
-    vocab_dir = vocab_params.pop('directory_path', None)
-
-    if vocab_dir is not None:
-        logger.info("Found a vocabulary.directory_path parameter in your config. "
-                    "Also saving the vocab we create to that location.")
-        os.makedirs(vocab_dir, exist_ok=True)
-
     os.makedirs(serialization_dir, exist_ok=True)
+    vocab_dir = os.path.join(serialization_dir, "vocabulary")
+
+    if os.path.isdir(vocab_dir) and os.listdir(vocab_dir) is not None:
+        raise ConfigurationError("The 'vocabulary' directory in the provided"
```

Comment:
Extra space needed here in the error message.

Reply:
Sure 👍